models/enterprise_sonic/br_l2pt/sonic_br_l2pt.yaml

```
@@ -0,0 +1,91 @@
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'community',
+        'license': 'Apache 2.0'
+    }
+NETWORK_OS: sonic
+RESOURCE: br_l2pt
+COPYRIGHT: Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved
+DOCUMENTATION: |
+    module: sonic_br_l2pt
+    version_added: '2.1.0'
+    short_description: Manage L2PT configurations on SONiC
+    description:
+      - This module provides configuration management of L2PT parameters
+        in devices running SONiC. 
+    author: 'allen.ting@dell.com'
```

```suggestion
    author: Allen Ting (allen.ting@dell.com)
```
If you don't mind putting your name

---

updated